ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/varpv6.yml

```diff
@@ -0,0 +1,52 @@
+# Testing duplicate VRF definitions according to the following rules documented in the schema.
+# Also see eos_designs_negative_unit_tests for testing of the raised errors.
+
+# It is recommended to only define a VRF in one Tenant. If the same VRF name is used across multiple tenants and those tenants
+# are accepted by `filter.tenants` on the node, any object set under the duplicate VRFs must either be unique or be an exact match.
+
+type: l3leaf
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 192.168.255.0/24
+    vtep_loopback_ipv4_pool: 192.168.254.0/24
+    virtual_router_mac_address: 00:dc:00:00:00:0a
+  nodes:
+    - name: varpv6
+      id: 101
+      bgp_as: 101
+      filter:
+        tenants:
+          - TENANT1
+
+tenants:
+  - name: TENANT1
+    mac_vrf_vni_base: 10000
+    vrfs:
+      - name: VRF1
+        vrf_id: 1
+        svis:
+          - id: 100
```

Comment:
can you add comment for each test about what you are testing? also you have twice the name `VARPv6_not_configured_2` for VLAN 200 and 400 - is it what you need here?

Reply:
added some comments